install.md

```diff
@@ -155,146 +146,77 @@ option.
     conda install graphviz python-graphviz pydotplus
```

Comment:
It seems odd to offer both `conda` and `virtualenv` instructions at one stage, and then require `conda` at the next. Should the should the notebook packages section be expanded to include alternatives? Perhaps we should only offer `conda` instructions?

Reply:
I will update my pull request on the base repository.